library/cc/engine_builder.cc

```diff
@@ -92,6 +92,35 @@ EngineBuilder& EngineBuilder::setDeviceOs(const std::string& device_os) {
   return *this;
 }
 
+EngineBuilder& EngineBuilder::setDecompressor(bool decompressor_on) {
+  this->decompressor_filter_ = decompressor_on;
+  return *this;
+}
+
+std::string EngineBuilder::generateFiltersStr() {
```

Comment:
I think with the alias reworking, this is pretty minimal YAML for adjusting the filter chain, WDYT?